src/Compilers/CSharp/Portable/Binder/Semantics/OverloadResolution/MethodGroup.cs

```diff
@@ -47,41 +47,15 @@ internal void PopulateWithExtensionMethods(
             this.PopulateHelper(receiverOpt, resultKind, error);
             this.IsExtensionMethodGroup = true;
 
-            PooledHashSet<MethodSymbol> implementationsToShadow = null;
```

Comment:
:memo: this deduplication between skeleton and implementation method is now handled upstream, in `EnumerateAllExtensionMembersInSingleBinder`